lib/plugin.js

```diff
@@ -230,13 +231,9 @@ plugin.emit = async function (div, item, done = () => {}) {
       })
       dialog.showModal()
       $('.close').on('click', () => dialog.close())
-      $('.retry').on('click', function () {
-        if (script.emit.length > 2) {
-          script.emit(div, item, () => done())
-        } else {
-          script.emit(div, item)
-          done()
-        }
+      $('.retry').on('click', async function () {
```

Comment:
This is maybe not that important, but I think both of these close and click handlers would be improved by providing the dialog element as context like so:
``` js
$('.close', dialog).on('click', ...)
$('retry', dialog).on('click', ...) 
```
I think this would only matter if there were multiple errors showing in a lineup, which is probably uncommon.

Reply:
I'm unable to find any jQuery documentation that explains what this second parameter is or means. This is as close as I come but this isn't what is being suggested here, I think.

https://api.jquery.com/context/

How would this second parameter be used? Are there further modifications required to make it work?

Thanks. Sorry to be so slow responding.

Reply:
`$('.close').on('click', ...)` adds a click handler that applies to every element in the entire `document` with the `close` class. By specifying the second parameter it reduces the scope from `document` to just the specific `dialog` element.

Reply:
Because we don't have many place in the code where `retry` or `close` classes are added, this is a pretty minor issue, I think.